index.ts

```diff
@@ -301,17 +303,20 @@ collect.set('isRepoPRList', [
 	'https://github.com/sindresorhus/refined-github/pulls?q=is%3Apr+is%3Aclosed',
 ]);
 
-// `issues/fregante` is a list but `issues/1`, `issues/new`, `issues/new/choose`, `issues/templates/edit` aren’t
 export const isRepoIssueList = (url: URL | HTMLAnchorElement | Location = location): boolean =>
-	Boolean(getRepo(url)?.path.startsWith('issues')) &&
-	!/^issues\/(\d+|new|templates)($|\/)/.test(getRepo(url)?.path!);
+	// `issues/fregante` is a list but `issues/1`, `issues/new`, `issues/new/choose`, `issues/templates/edit` aren’t
+	/^issues(?!\/(\d+|new|templates)($|\/))/.test(getRepo(url)?.path!) ||
+	Boolean(getRepo(url)?.path.startsWith('labels/'));
```

Comment:
Wont this match `https://github.com/sindresorhus/refined-github/labels/`?

Reply:
No, because `getRepo()` uses `getCleanPathname()` which strips the trailing slash.

Reply:
Lets add a new list of "not-yet-matched" URLs somewhere so we don't have to guess, like this at the beginning of the file:

```js
collect.set('__urls_that_dont_match__', [

]);
```

It can also include the URLs specified in the comment, unless they already match something:

```
// `issues/fregante` is a list but `issues/1`, `issues/new`, `issues/new/choose`, `issues/templates/edit` aren’t
```